docs/reference/search/field-stats.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[search-field-stats]]
+== Field stats API
+
+experimental[]
+
+The field stats api allows to easily figure certain statistical properties without a field without executing a search, but
+looking up measurements that are natively available in the Lucene index. This can be useful to explore a data set,
+that you don't know much about in order to for example create histogram aggregation with a meaningful interval.
```

Comment:
I would rework the second sentence into this:
`This can be useful to explore a dataset which you don't know much about. For example, this allows creating a histogram aggregation with meaningful intervals`